src/pages/index.js

```diff
@@ -50,20 +51,20 @@ const VaxLocation = (location) => {
       border={"1px solid black"}
       alignItems="start"
       w={["100vw", "30vw"]}
-      h={["10em"]}
+      minHeight={["10em"]}
     >
       <Stack padding={1} w="100%">
         <Text>{namaLokasi}</Text>
         <Text>
           KEC/KEL: {kecamatan} / {kelurahan}
         </Text>
         <Text>{wilayah}</Text>
-        <Stack direction="row">
+        <Stack direction="row" wrap="wrap" gridRowGap={2} paddingBlockEnd={2}>
           {jadwal.map(({ id, waktu }) => {
             return (
               <Popover key={id}>
                 <PopoverTrigger>
-                  <Button>{id}</Button>
+                  <Button margin={0}>{id}</Button>
```

Comment:
whoops, seems like margin-left in this button doesn't work, will revert

Reply:
Eh ok will hold the erge

Reply:
done